subprojects/packagefiles/glad/meson.build

```diff
@@ -0,0 +1,9 @@
+project('glad',
+    'c',
+    version: 'v0.1.36',
+    license: 'MIT'
+)
+
+glad_dep = declare_dependency(
+    include_directories: include_directories('.')
+)
```

Comment:
From a quick look at the glad project, you need to actually generate all headers. Their cmake needs to be converted to Meson.